packages/frontend/src/context/User.ts

```diff
@@ -365,7 +364,7 @@ export class User {
             currentEpoch,
             subsidyKey
         )
-        if (spentSubsidy >= this.unirepConfig.subsidy) {
+        if (spentSubsidy > 0) {
```

Comment:
you should use `spentSubsidy.toNumber() > 0` like I said in https://github.com/Unirep/Unirep-Social/pull/181#discussion_r1049190497
since `spentSubsidy` is `ethers.BigNumber` type so it is an object not a `number`
it always returns false if comparing to a number
I tried `spentSubsidy === 0` that returns `false` if it hasn't got airdrop
but `spentSubsidy.toNumber() === 0` returns `true` if it hasn't got airdrop